sdk/program-option/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "solana-program-option"
+description = "A C representation of Rust's Option, used in Solana programs."
+documentation = "https://docs.rs/solana-program-option"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dev-dependencies]
+# circular dev deps need to be path deps for `cargo publish` to be happy
+solana-program = { path = "../program" }
```

Comment:
What's this about? Is it needed because of the doctests? Can we change the doctests to use the new crate path instead? Ie `use solana_program_option::COption`?

Reply:
Oh yeah for this crate case we can just do that